### ISSUE TITLE
glfwCreateWindow crashes on Windows 8.1

### ISSUE THREAD

**Issue by ghuser404** [glfw/glfw#1335]
Because `GetDpiForWindow` is `NULL`.

```
getFullWindowSize(getWindowStyle(window), getWindowExStyle(window),
                  0, 0, &xoff, &yoff,
                  GetDpiForWindow(window->win32.handle));
```

**Comment by MrROBUST** [comment]
I have this issue on Win7 x64. The regression is probably here: 089ea9af227fdffdf872348923e1c12682e63029

**Comment by ghuser404** [comment]
No, it's here: [`b3efdcb`](https://github.com/glfw/glfw/commit/5294439595e5b95c238a121c8a63c076a0aa2c8a)

**Comment by cdwfs** [comment]
It looks like all the GetDpi*() functions are Win10+. I'm not sure what the equivalent approach would be for Win7/Win8; [this thread](https://social.msdn.microsoft.com/Forums/windows/en-US/8aa4efbc-07ce-4c4f-91a3-d25b62606136/how-can-i-programmatically-get-the-current-dpi-setting-from-windows) looked potentially promising, but I don't have a moment to dig into it right now.

**Comment by elmindreda** [comment]
@ghuser404 Thank you!

@cdwfs On versions of Windows where `GetDpiForWindow` isn't available that argument isn't used anyway, so your solution is fine.

**Comment by cdwfs** [comment]
Works for me! :) I just wasn't sure if it was worth going the extra mile to figure out the equivalent code to extract the DPI scaling in pre-Win10, if it even exists.

**Comment by elmindreda** [comment]
It does! We do it for the monitor and window content scale queries added recently.
